_layouts/home.html

```diff
@@ -12,14 +12,52 @@ <h2 id="countdown">&nbsp;</h2>
       This campaign is... something!
     </p>
   </div>
+  
+              <div class="carousel-inner" role="listbox">
```

Comment:
can you make sure that this whole part starts at the same indentation as the div above it? (i.e. 2 spaces after the beginning of the line)